app/src/main/res/layout/activity_change_contact.xml

```diff
@@ -9,7 +9,7 @@
     tools:context="com.nervousfish.nervousfish.activities.ContactActivity">
 
     <android.support.design.widget.AppBarLayout
-        android:id="@+id/app_bar"
+        android:id="@+id/app_bar_change_contact"
```

Comment:
I like the name change but I would suggest to put the `change_contact` part at the beginning of the ID so all ID's related to the change contact part start the same